src/developer/web-api/tracker.md

```diff
@@ -3425,3 +3425,207 @@ All merging is based on data already persisted in the database, which means the
+
+NOTE: WEB_HOOK notificationRecipient is used only to POST http request to an external system. Make sure to choose HTTP delivery channel when using WEB_HOOK.
+
+## Retrieving and deleting Program Notification Template
+
+As ProgramNotificationTemplate is a type of metadata, you can create, update, and delete it just like other metadata.
+
+### Program Messages
+
+The program message feature enables you to send messages to tracked entity instances, 
+contact addresses associated with organizational units, phone numbers, and email addresses. 
+Messages can be sent using the messages resource.
+
+    /api/messages
+
+### Sending program messages
+
+Program messages can be sent using two delivery channels:
+
+  - SMS (SMS)
+
+  - Email address (EMAIL)
+
+Program messages can be sent to various recipients:
+
+  - Tracked entity: The system will look up attributes of value
+    type PHONE_NUMBER or EMAIL (depending on the specified delivery
+    channels) and use the corresponding attribute values.
+
+  - Organisation unit: The system will use the phone number or email
+    information registered for the organisation unit.
+
+  - List of phone numbers: The system will use the explicitly defined
+    phone numbers.
+
+  - List of email addresses: The system will use the explicitly defined
+    email addresses.
+
+Below is a sample JSON payload for sending messages using POST requests.
+Note that message resource accepts a wrapper object named
+`programMessages` which can contain any number of program messages.
+
+    POST /api/33/messages
+
+```json
+{
+  "programMessages": [{
+    "recipients": {
+      "trackedEntity": {
+        "id": "UN810PwyVYO"
+      },
+      "organisationUnit": {
+        "id": "Rp268JB6Ne4"
+      },
+      "phoneNumbers": [
+        "55512345",
+        "55545678"
+      ],
+      "emailAddresses": [
+        "johndoe@mail.com",
+        "markdoe@mail.com"
+      ]
+    },
+    "enrollment": {
+      "id": "f3rg8gFag8j"
+    },
+    "event": {
+      "id": "pSllsjpfLH2"
+    },
+    "deliveryChannels": [
+      "SMS", "EMAIL"
+    ],
+    "notificationTemplate": "Zp268JB6Ne5",
+    "subject": "Outbreak alert",
+    "text": "An outbreak has been detected",
+    "storeCopy": false
+  }]
+}
+```
+
+The fields are explained in the following table.
+
+
+
+Table: Program message payload
+
+| Field | Required | Description | Values |
+|---|---|---|---|
+| recipients | Yes | Recipients of the program message. At least one recipient must be specified. Any number of recipients / types can be specified for a message. | Can be trackedEntity, organisationUnit, an array of phoneNumbers or an array of emailAddresses. |
+| enrollment | Enrollment which ProgramMessage is attached to | Enrollment. | Enrollment ID. |
+| event      | Event which ProgramMessage is attached to | Event. | Event ID. |
+| deliveryChannels | Yes | Array of delivery channels. | SMS &#124; EMAIL |
```

Comment:
Why is the pipe different here? Or was it not pipe?

Reply:
i think it says 2 possible values` SMS & EMAIL`